src/tests/ftest/soak/smoke.yaml

```diff
@@ -235,7 +234,40 @@ macsio_smoke:
     mount_dir: "/tmp/daos_dfuse/macsio/"
     disable_caching: true
     thread_count: 8
-    cores: '0-3'
+    cores: '0-7'
+datamover_smoke:
+  job_timeout: 10
+  nodesperjob:
+    - 1
+  taskspernode:
+    - 1
+  oclass:
+    - ["EC_2P1GX", "RP_2GX"]
+  dcp:
+    bufsize: 4M
+    chunksize: 128M
```

Comment:
FYI in case `dcp --help` isn't clear:
- bufsize - The IO read/write size. Similar to IOR transfer_size
- chunksize - The size of each chunk of work. In this case, 128M means the file is split into chunks of 128M and each process gets an equal number of chunks.

Caveat to `chunksize` by example:
```
mpirun -np 4 ior -w -k -t 1M -b 1M ...
# Creates file of size 4M - no problem
mpirun -np 4 dcp  --chunksize 128M ...
# File is split into chunks of 128M
# rank0 works on 128M
# ranks 1,2,3 work on nothing
```